bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/targetComponent/TargetFeedback.scala

```diff
@@ -207,7 +208,9 @@ object BagsFeedback {
     case RunningState(_,_,_)   => RunningStateRow
     case RunningEditedState(_) => RunningStateRow
     case FailureState(_,why)   => FailureStateRow(why)
-    case IdleState(_,_) if ctx.exists(_.getTargets.getGuideEnvironment.guideEnv.auto === AutomaticGroup.Initial) => NoStarsRow
+    case IdleState(_,_) if ctx.exists(c =>
+      AgsRegistrar.currentStrategy(c).exists(_.key.id =/= OffStrategy.key.id) &&
```

Comment:
I was about to create a proper Eq instance but thought it might not be worth the trouble if using the AgsStrategy.Key is good enough.